Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -295,6 +294,33 @@ def IccLeftChart (x y : ℝ) [h : Fact (x < y)] :
     have B : Continuous fun z : EuclideanSpace ℝ (Fin 1) => z 0 := continuous_apply 0
     exact (A.comp B).comp continuous_subtype_val
 
+variable {x y : ℝ} [hxy : Fact (x < y)]
+
+namespace Fact.Manifold
+
+scoped instance : Fact (x ≤ y) := Fact.mk hxy.out.le
+
+end Fact.Manifold
+
+open Fact.Manifold
+
+lemma IccLeftChart_extend_left_eq :
+    ((IccLeftChart x y).extend (𝓡∂ 1)) (Icc.instOrderBotElem.bot) = 0 := by
+  calc ((IccLeftChart x y).extend (𝓡∂ 1)) (Icc.instOrderBotElem.bot)
+    _ = (𝓡∂ 1) ⟨fun _ ↦ 0, by norm_num⟩ := by norm_num [IccLeftChart, Icc.coe_bot]
+    _ = 0 := rfl
```

Comment:
Seems like there's a lot of `rfl`s doing the heavy lifting. Are we missing API?

Reply:
I've fixed all but three rfls; those seem to be missing API/ I certainly can't find them. Help finding those is welcome.

Reply:
I have added these lemmas as helper results now.